services/app-graphql/src/queries/indexRates.graphql

```diff
@@ -74,6 +74,71 @@ query indexRates {
                             packageId
                             packageStatus
                         }
+                    }
+                                     contractRevisions {
+                        id
+                        contract {
+                            id
+                            stateCode
+                            stateNumber
+                        }
+                        createdAt
+                        updatedAt
+                        submitInfo {
+                            updatedAt
+                            updatedBy
+                            updatedReason
+                        }
+                        unlockInfo {
+                            updatedAt
+                            updatedBy
+                            updatedReason
+                        }
+                        formData {
+                            programIDs
+                            populationCovered
+                            submissionType
+                            riskBasedContract
+                            submissionDescription
+                            stateContacts {
+                                name,
+                                titleRole
+                                email
+                            }
+                            supportingDocuments {
+                                name
+                                s3URL
+                                sha256
+                            }
+                            contractType
+                            contractExecutionStatus
+                            contractDocuments {
+                                name
+                                s3URL
+                                sha256
+                            }
+                            contractDateStart
+                            contractDateEnd
+                            managedCareEntities
+                            federalAuthorities
+                            inLieuServicesAndSettings
+                            modifiedBenefitsProvided
+                            modifiedGeoAreaServed
+                            modifiedMedicaidBeneficiaries
+                            modifiedRiskSharingStrategy
+                            modifiedIncentiveArrangements
+                            modifiedWitholdAgreements
+                            modifiedStateDirectedPayments
+                            modifiedPassThroughPayments
+                            modifiedPaymentsForMentalDiseaseInstitutions
+                            modifiedMedicalLossRatioStandards
+                            modifiedOtherFinancialPaymentIncentive
+                            modifiedEnrollmentProcess
+                            modifiedGrevienceAndAppeal
+                            modifiedNetworkAdequacyStandards
+                            modifiedLengthOfContract
+                            modifiedNonRiskPaymentArrangements
```

Comment:
This seems to be missing new fields I added to contractFormData in a recentish PR for `statutoryRegulatoryAttestationDescription` and `statutoryRegulatoryAttestation`